mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -93,6 +94,43 @@ def gridded_inverse_fourier_transform_nd(kspace_loc,
         pfft.ifftn(pfft.ifftshift(gridded_kspace))), 1, 0)
 
 
+def gridded_inverse_fourier_transform_stack(plane_samples, z_samples,
+                                            stack_len, kspace_data, grid,
+                                            method):
+    """
+    This function calculates the gridded Inverse fourier transform
+    from Interpolated non-Cartesian data into a cartesian grid. However,
+    the IFFT is done similar to Stacked FOurier transform.
+
+    Parameters
+    ----------
+    kspace_loc: np.ndarray
+        The N-D k_space locations of size [M, N]
+    kspace_data: np.ndarray
+        The k-space data corresponding to k-space_loc above
+    method: {'linear', 'nearest', 'cubic'}, optional
+        Method of interpolation for more details see scipy.interpolate.griddata
+        documentation
+    grid: np.ndarray
+        The Gridded matrix for which you want to calculate k_space Smaps
+    Returns
+    -------
+    np.ndarray
+        The gridded inverse fourier transform of given kspace data
+    """
+    gridded_kspace = []
+    for i in np.arange(len(z_samples)):
+        gridded_kspace.append(
+            griddata(plane_samples,
+                     kspace_data[i*stack_len:(i+1)*stack_len],
+                     grid,
+                     method=method,
+                     fill_value=0))
+    gridded_kspace = np.moveaxis(np.asarray(gridded_kspace), 0, 2)
```

Comment:
Could you add a comment on all these changes of axis (`moveaxis`, `swapaxes`)?

Reply:
Yes on it